src/components/Modal/ToDoModal/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import Image from 'next/image';
+import styles from './ToDOModal.module.scss';
+import ProfileIcon from '../../Profile/ProfileIcon';
+import ModalDropdown from '../ModalDropdown';
+import Modal from '../Modal';
+
+type ModalProps = {
+  showModal: boolean;
+  handleClose: () => void;
+};
+
+function ToDoModal({ showModal, handleClose }: ModalProps) {
+  const [isDropdown, setIsDropdown] = useState(false);
+
+  const handleDropdownOpen = () => {
+    setIsDropdown(true);
+  };
+
+  const handleDropdownClose = () => {
+    setIsDropdown(false);
+  };
+
+  return (
+    <Modal showModal={showModal} handleClose={handleClose}>
+      <div className={styles.btnBox}>
+        <div className={styles.btns}>
+          <button onClick={handleDropdownOpen} className={styles.btnx}>
+            <Image
+              src="/svgs/kebab.svg"
+              alt="케밥 버튼"
+              width={28}
+              height={28}
+            />
+          </button>
+          {isDropdown && (
+            <ModalDropdown
+              showDropdown={isDropdown}
+              handleDropdownClose={handleDropdownClose}
+            />
+          )}
+          <button onClick={handleClose} className={styles.btnx}>
+            <Image src="/svgs/close.svg" alt="닫기" width={28} height={28} />
+          </button>
+        </div>
+        <div className={styles.name}>새로운 일정 관리 Taskify</div>
+      </div>
+      <div className={styles.contentBox}>
+        <div className={styles.managerBox}>
+          <div className={styles.nameBox}>
+            <div className={styles.manager}>담당자</div>
+            <div className={styles.managerProfile}>
+              <ProfileIcon small />
+              <div className={styles.managerName}>정만철</div>
+            </div>
+          </div>
+          <div>
+            <div className={`${styles.manager} ${styles.doneDate}`}>마감일</div>
+            <div className={styles.managerName}>2022.12.30 19:00</div>
+          </div>
+        </div>
+        <div>
+          <div className={styles.content}>
+            <div className={styles.text}>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              Vestibulum finibus nibh arcu, quis consequat ante cursus eget.
+              Cras mattis, nulla non laoreet porttitor, diam justo laoreet eros,
+              vel aliquet diam elit at leo.
+            </div>
+            <div className={styles.img}>
+              <Image
+                src="/svgs/example.svg"
+                alt="예시 사진"
+                layout="responsive"
```

Comment:
```suggestion
                layout="responsive"
```
요거 layout 속성 사용하면 경고 안나오시나요?? 저도 메인 페이지할 때 layout 사용했었는데 경고나와서 찾아보니깐 next.js 구버전에서 사용하고 요즘은 권장하지 않는 방법이라고 하더라구요!